### ISSUE TITLE
pianoKey label blocks interaction

### ISSUE THREAD

**Issue by fideldonson** [AppleEducate/flutter_piano#4]
The key label blocks interaction with the button beneath. An easy fix is to wrap the label Text widget in an IgnorePointer widget.

Thanks for the great work btw.

**Comment by rodydavis** [comment]
Absolutely! Should be resolved on the latest update. Let me know if it isn't
